plugins/@grouparoo/demo/src/util/shared.ts

```diff
@@ -9,15 +11,17 @@ export async function execSync(command) {
   await sharedExecSync(command);
 }
 
-export async function prettier(fileOrDirPath) {
-  // prettier format
-  const pCmd = path.resolve(
-    path.join(__dirname, "..", "..", "node_modules", ".bin", "prettier")
-  );
-  const pConfig = path.resolve(
-    path.join(__dirname, "..", "..", "..", "..", "..", ".prettierrc")
-  );
-  await execSync(`'${pCmd}' --config '${pConfig}' --write '${fileOrDirPath}'`);
+export async function prettier(fileOrDirPath: string) {
+  const stats = fs.statSync(fileOrDirPath);
+  if (stats.isDirectory) {
+    const files = glob.sync(`${fileOrDirPath}/**/*.{json,js}`);
+    for (const file of files) await prettier(file);
+  } else {
+    const text = fs.readFileSync(fileOrDirPath).toString();
+    const pOptions = await Prettier.resolveConfig(process.cwd());
+    const formatted = Prettier.format(text, pOptions);
+    fs.writeFileSync(fileOrDirPath, formatted);
+  }
```

Comment:
Rather than using the prettier binary, we can use it pragmatically.  This allows us to not worry about the install path - loading prettier as a dependency just works!

Reply:
seemed pretty predicable before and I trusted their recursiveness more than my own, but open

Reply:
The path to the `prettierrc` doesn't work if you are outside of the monorepo (eg you installed this from NPM) - that was the main change